coroutine.cc

```diff
@@ -307,6 +307,7 @@ void Coroutine::AddPollFds(std::vector<struct pollfd> &pollfds,
                            std::vector<Coroutine *> &covec) {
   switch (state_) {
   case State::kCoReady:
+    [[fallthrough]];
```

Comment:
That's an annoying warning to be sure.  But no problem if it's needed

Reply:
Part of MISRA-23 and Autosar-14. And I kind of like the explicit fallthrough thing.

Reply:
Only if there is code between the case labels.  When they are adjacent it is clearly not a coding error.  i wish they would add "case 1;2;3:" to C++.